platform.md

```diff
@@ -845,7 +846,7 @@ paths = ["<app sub-path glob>"]
 ```
 
 Where:
-- Both `id` and `version` MUST be present for each buildpack.
+- `id`, `version` and `api` MUST be present for each buildpack.
```

Comment:
```suggestion
- `id`, `version`, and `api` MUST be present for each buildpack.
```

Reply:
I am glad the CNB spec has an opinion on the oxford comma ;)